.golangci.yml

```diff
@@ -65,6 +65,10 @@ issues:
       linters:
         - gosec
         - forbidigo
+    # gci is confused by "slices" and suggests to move it to the end of the import list
+    - path: tools/protovalidate-conformance/internal/results/result.go
+      linters:
+        - gci
```

Comment:
Without ignoring it, running golangci-lint with `--fix` would update this file's import list to 
```
    "cmp"
	"github.com/bufbuild/protovalidate/tools/internal/gen/buf/validate"
	"github.com/bufbuild/protovalidate/tools/internal/gen/buf/validate/conformance/harness"
	"slices"
```